key/key.go

```diff
@@ -0,0 +1,12 @@
+package key
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+
+	"github.com/CodeChain-io/codechain-rpc-go/crypto"
+)
+
+func GenerateKey() (*ecdsa.PrivateKey, error) {
```

Comment:
I'm not confident about the return type.
Should we return the private key as byte[] type or PrivateKey type?

@cubismic 
What do you think about it?

Reply:
I think []byte type would be better because we don't use PrivateKey type in other codes.